ember.md

```diff
@@ -67,6 +67,28 @@ foo: undefined
 foo: null
 ```
 
+<a name="general-structure--multiProperty"></a>
```

Comment:
These anchor tags aren't needed in markdown. You can check the JavaScript standard for examples of how to handle linking native to markdown.

Reply:
@ASH-Michael Should we consider adding a issue for that?